metadata.go

```diff
@@ -760,6 +879,37 @@ func getTableMetadata(session *Session, keyspaceName string) ([]TableMetadata, e
 	return tables, nil
 }
 
+// query for only the table metadata in the specified keyspace from system_virtual_schema.tables
+func getVirtualTableMetadata(s *Session, keyspaceName string) ([]VirtualTableMetadata, error) {
+	const stmt = `
+		SELECT
+				table_name,
+				comment
+			FROM system_virtual_schema.tables
+			WHERE keyspace_name = ?`
```

Comment:
Can you please fix indent:
```suggestion
	const stmt = `
		SELECT
			table_name,
			comment
		FROM system_virtual_schema.tables
		WHERE keyspace_name = ?`
```

Reply:
agree, fixed